File: 4st/Eunseong/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;

        while (order > 0) {
            if (order % 10 % 3 == 0 && order % 10 != 0) answer++;
            order /= 10;
        }

        return answer;
    }
}

File: 3st/Juram/programmer/369게임.js
function solution(order) {
    var answer = 0;
    order = order.toString();
    var arr = order.split("");
    for(let i=0; i<arr.length; i++){
        if(arr[i]=="3" || arr[i]=="6" || arr[i]=="9"){
            answer+=1;
        }
    }
    return answer;
}